backends/arm/arm_partitioner.py

```diff
@@ -158,3 +160,17 @@ def filter_fn(node: torch.fx.Node) -> bool:
         ] + ops_to_not_decompose_if_quant_op
 
         return (ops_to_not_decompose, filter_fn)
+
+
+class ArmEthosUPartitioner(ArmTOSAPartitioner):
```

Comment:
Nit: different file? I would say we should create different directories i.e 
```
└── backend
    └── arm
        ├── ethos-u
        │   ├── backend.py
        │   ├── partitioner.py
        │   └── quantizer.py
        ├── tosa
        │   ├── backend.py
        │   ├── partitioner.py
        │   └── quantizer.py
        └── <something_else>
            ├── backend.py
            ├── partitioner.py
            └── quantizer.py
```

Again, what you have is fine for now, if we plan ahead a bit, it will scale nicely.